Runtime/Scripts/InstanceRenderingInfo.cs

```diff
@@ -0,0 +1,238 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Rendering;
+
+namespace Unity.MeshSync
+{
```

Comment:
[Serializable] not required ?

Reply:
There was an iteration where InstanceRenderingInfo was serializable but we have moved away from that